pkg/nuctl/test/get_test.go

```diff
@@ -74,29 +78,55 @@ func (suite *GetTestSuite) TestGet() {
 
 		suite.Require().NoError(err)
 
-		// cleanup
+		// cleanup - try delete
+		// make sure to clean up after the test
+		// use nutctl to delete the function when we're done
 		defer func() {
 
 			// make sure to clean up after the test
-			suite.dockerClient.RemoveImage(imageName)
+			suite.dockerClient.RemoveImage(functionImages[0])
+
+			// use nutctl to delete the function when we're done
+			suite.ExecuteNutcl([]string{"delete", "fu", functionNames[0]}, nil)
+
+			suite.dockerClient.RemoveImage(functionImages[1])
+			suite.dockerClient.RemoveImage(functionImages[2])
 
 			// use nutctl to delete the function when we're done
-			suite.ExecuteNutcl([]string{"delete", "fu", functionName}, nil)
+			suite.ExecuteNutcl([]string{"delete", "fu", functionNames[1], functionNames[2]}, nil)
 		}()
 	}
 
 	err := suite.ExecuteNutcl([]string{"get", "function"}, nil)
 
 	suite.Require().NoError(err)
 
-	foundFunctions := make([]bool, len(functionNames))
+	foundFunctionTests := [][]bool{getCheckOutputTest(suite, functionNames)}
+
+	multiplyGetTestFunctions := []string{functionNames[0], functionNames[1]}
+	err = suite.ExecuteNutcl([]string{"get", "fu", multiplyGetTestFunctions[0], multiplyGetTestFunctions[1]}, nil)
+
+	suite.Require().NoError(err)
+
+	foundFunctionTests = append(foundFunctionTests, getCheckOutputTest(suite, multiplyGetTestFunctions))
+
+	for _, foundFunctions := range foundFunctionTests {
+		for _, foundFunction := range foundFunctions {
+			suite.Require().True(foundFunction)
+		}
+	}
+}
+
+func getCheckOutputTest(suite *GetTestSuite, requiredNames []string) []bool {
```

Comment:
Document this function: explain what it receives, what it returns and how it works.